pkg/reader/ccip.go

```diff
@@ -207,6 +207,32 @@ type RMNCurseResponse struct {
 
 // ---------------------------------------------------
 
+// printReports is used to trim the size of the printed report. There can be a
+// large number of reports, especially on Solana where gas and token
+// price updates are split into separate reports. This function removes price
+// only reports, and removes price data from combined reports in an attempt to
+// ensure merkle roots are easier to find if logs are truncated.
+func printReports(lggr logger.Logger, reports []cciptypes.CommitPluginReportWithMeta) {
+	tokenPriceUpdates := 0
+	gasPriceUpdates := 0
+	var truncatedReports []cciptypes.CommitPluginReportWithMeta
```

Comment:
don't think this function cares whether they're truncated or not.
```suggestion
	var reportToPrint []cciptypes.CommitPluginReportWithMeta
```